src/docs/predicted_molecular_consequence.md

```diff
@@ -5,12 +5,91 @@ A set of information enabling interpretation of likely variant impact on genomic
 | Field             | Type            | Description
 |-------------------|-----------------|---------------------
 | allele_name           | string          | Data-derived allele name
-| feature_stable_id | string          | Feature (eg transcript) id
+| stable_id | string          | Feature stable id (transcripts for now)
 | feature_type      | OntologyTermMetadata        | SO term for feature type (eg. transcript, regulatory feature)
 | consequences      | array of OntologyTermMetadata           | SO terms for predicted impact ( eg missense variants)
-| variant_representation            | array of VariantRepresentation or [ ]  | Nomenclature values and source
+| gene_stable_id | string| Gene stable id
+| gene_symbol | string | Gene symbol
+| protein_stable_id  | string or null | Protein stable id
+| variant_representation | array of VariantRepresentation or [ ]  | Nomenclature values and source
 | prediction_results | array of PredictionResult or [ ] | Scores from programs like SIFT which calculate transcript-specific deleteriousness scores
```

Comment:
Check out the description of the `PredictionResult` type. Its `classification` field is described as being of a `ValueSet` type; while in your example json, it is a string. Either the type description or the example will need updating.

Reply:
Yes @azangru , this is incorrect in the json, I will update it to be `ValueSet`.